### ISSUE TITLE
Is there anyway to debug this?

### ISSUE THREAD

**Issue by AlgorithmsAreCool** [dgiagio/warp#5]
A minimal example works fine, but when I tried on a larger application i got

```
Specify --help for a list of available options and commands.
Unrecognized command or argument 'MyApp.exe'
```

But I can run the app just fine from the warp packages folder in appdata.

Can I get some more verbose output somehow to try to figure out whats wrong?

**Comment by dgiagio** [comment]
A bug that might have affected your case has just been fixed (#2). Could you please download the new release v0.1.1 and try again?

About your question: if you set the environment variable `WARP_TRACE=1` you get debugging output while running your self-contained application.

**Comment by AlgorithmsAreCool** [comment]
BRILLIANT! It works!
Thank you so much for this project!
